<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExamDialog</class>
 <widget class="QDialog" name="AddExamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddExamDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="addTWButton">
       <property name="text">
        <string>Письменный</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="addAbButton">
       <property name="text">
        <string>Тестовый</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="rightALabel">
       <property name="text">
        <string>&lt;No item selected&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Добавить вопросы:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="9" colspan="2">
      <widget class="QListWidget" name="questList"/>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="examEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
